### PR TITLE
docs: update slot

### PR DESCRIPTION
**Pull request by mhevery**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/(qwik)/components/state/index.mdx [hunk 1]
```diff
@@ -492,28 +487,23 @@ If a value is marked as non-serializable then that value will not survive serial
...
-  type NoSerialize,
 } from '@builder.io/qwik';
-import * as monaco from 'monaco-editor';
+import Monaco from './monaco';
 
 export default component$(() => {
-  const editorRef = useSignal<Element>();
-  const store = useStore<{ monacoInstance: NoSerialize<Monaco> }>({
+  const store = useStore<{ monacoInstance: Monaco | undefined }>({
+    // Don't initialize on server
```
Comment (manucorporat) [inline packages/docs/src/routes/docs/(qwik)/components/state/index.mdx:497]: why? can you revert?

**Review by manucorporat — DISMISSED** [review]
Added some comments

File: packages/docs/src/routes/docs/(qwik)/components/state/index.mdx (hunk 1 above)
Comment (mhevery) [reply packages/docs/src/routes/docs/(qwik)/components/state/index.mdx:497]: opps, will revert

File: packages/docs/src/routes/docs/(qwik)/getting-started/index.mdx [hunk 2]
```diff
@@ -111,7 +111,6 @@ export const useDadJoke = routeLoader$(async () => {
 });
 
 export default component$(() => {
-  // Calling our `useDadJoke` hook, will return a reactive signal to the loaded data.
```
[the remark this answers is not in the document]
Comment (mhevery) [reply packages/docs/src/routes/docs/(qwik)/getting-started/index.mdx:114]: opps, will revert

**Pull request merged**